### ISSUE TITLE
:sourceType and :source_type error with oci gem > 2.0.9

### ISSUE THREAD

**Issue by ehaselwanter** [stephenpearson/kitchen-oci#4]
```
>>>>>>     Failed to complete #create action: [You cannot provide both :sourceType and :source_type] on default-oracle
```

**Comment by stephenpearson** [comment]
Thank you.  It seems that the OCI ruby API has changed in 2.1.0.  I will submit a branch to update the API call after doing some testing, and I'll pin the OCI release in the Gemspec.

Basically you just need this to unblock yourself:
```ruby
      def instance_source_details(config)
        OCI::Core::Models::InstanceSourceViaImageDetails.new(
          sourceType: 'image',
          imageId: config[:image_id]
        )
      end
```

**Comment by stephenpearson** [comment]
This is now fixed in version 1.2.0 of the gem which was merged in #6.  Please confirm whether that works for you.

**Comment by ehaselwanter** [comment]
I can confirm that it works for private networks.

```shell
kitchen -v
Test Kitchen version 1.20.0

chef gem list |grep oci
kitchen-oci (1.2.0)
oci (2.1.0)
```
